### PR TITLE
added selu function to jax frontend nn and its tests

### LINKED ISSUE #4919: selu

### PR DESCRIPTION
**Pull request by kundakinte**
Close #4919

### CODE REVIEW

**Comment by kundakinte** [comment]
hey @Auxia, sorry if you’re busy, may you review my code please

**Comment by kundakinte** [comment]
hey @Auxia ,if you aren’t too busy could you please review my code, thank you :)

**Comment by Auxia** [comment]
Hi @kundakinte, sorry for the late reply! Is this an ivy extension request or a frontend implementation? Afaik, we do not have a an implementation for `selu` at the backend. For the frontend task it's required to use the `ivy.<backend>` functions in the implementations. If you want to work on an extension task then you can look for one in `extensions` label.

Thanks!

**Comment by kundakinte** [comment]
hello @Auxia, no worries. this is a Jax front end implementation, one of the 23 tasks in the “Add Non-Linear activation functions to JAX frontend”. I have made changes and used ivy.backend functions

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by kundakinte** [comment]
Hello I’m waiting for my pull request to be reviewed 
If you’re free @Auxia or anybody else could you please review it

**Comment by Auxia** [comment]
Hi @kundakinte, There seem to be some conflicts that are yet to be resolved. Also no checks have been run on this PR, which is pretty peculiar. Can you resolve the conflicts and then I can quickly get it reviewed this weekend. I will also figure out why no checks have been run on your latest commit.

**Comment by kundakinte** [comment]
Hey @Auxia , sorry, saw this late, resolved the conflicts, ready for review

**Comment by kundakinte** [comment]
waiting for a code review

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by kundakinte** [comment]
hello, still waiting for feedback from @Auxia

**Comment by kundakinte** [comment]
hey @ogbanugot , waiting for feedback

**Comment by ogbanugot** [comment]
Also, the tests are not passing, you might want to fix that if you do want to contribute the current implementation you have here.

**Comment by ogbanugot** [comment]
So, I think following the implementation [here](https://jax.readthedocs.io/en/latest/_modules/jax/_src/nn/functions.html#selu) this works;
```
@to_ivy_arrays_and_back
def selu(x):
    alpha = 1.6732632423543772848170429916717
    scale = 1.0507009873554804934193349852946
    return scale * elu(x, alpha)
```

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py [hunk 1]
```diff
@@ -822,3 +822,37 @@ def test_jax_nn_hard_sigmoid(
         on_device=on_device,
         x=xs[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.nn.selu",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (ogbanugot) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_nn_activations.py:830]: should test for `valid` here not just `float`

**Comment by ogbanugot** [comment]
If you use the implementation above, plus the small change to the test, everything works just fine. Unless you have an explanation for the implementation you have currently, then feel free to go ahead and make the suggested changes and then request a review from me when done. Thanks!

**Comment by kundakinte** [comment]
hello @ogbanugot , thank you so much for the review. funnily enough seems like my initial implementation was correct, you can see it in my earlier commits, but i was made to believe there was another way. anyways i reverted back to my initial implementation and changed float to valid. thank you

**Review by ogbanugot — APPROVED** [review]
